docs/settings.rst

```diff
@@ -103,6 +103,29 @@ of ``mozilla-django-oidc``.
 
    The OpenID Connect scopes to request during login.
 
+.. py:attribute:: OIDC_STORE_ACCESS_TOKEN
+
+   :default: ``False``
+
+   Controls whether the OpenID Connect client stores the OIDC ``access_token`` in the user session.
+   The session key used to store the data is ``oidc_access_token``.
+
+   By default we want to store as less credentials as possible so this feature defaults to ``False``
+   and is discouraged to use.
+
+   .. danger::
+      Use at your own risk. By enabling this feature you are storing OIDC credentials to the user session.
+      Don't combine it with Django's cookie-based session backend because this will have as a result leaking
+      OIDC credentials to the client side.
```

Comment:
I think "Use at your own risk" is a little strong here.  The client side is better known as the "user agent" so it's not the worst place to have "user credentials"!  In fact, there are OIDC flows where these tokens are *only* stored on the user agent.  The rest of the note is accurate, so perhaps reword to:

> .. warning::
>     This feature stores authentication information in the session.  If used in combination with Django's [cookie-based session backend](https://docs.djangoproject.com/en/1.11/topics/http/sessions/#using-cookie-based-sessions), those tokens will be visible in the browser's cookie store.